File: 8-kyu/sum-the-strings/javascript/solution.js
const sumStr = (a, b) => {
  let aInt = parseInt(a);
  if (!aInt) {
    aInt = 0;
  }
  let bInt = parseInt(b);
  if (!bInt) {
    bInt = 0;
  }
  const result = aInt + bInt;
  return result.toString();
}

File: 8-kyu/sum-the-strings/swift/solution.swift
func sum_str(_ a: String, _ b: String) -> String {
    if let aInt = Int(a) {
        if let bInt = Int(b) {
            return "\(aInt + bInt)"
        }
        return "\(aInt)"
    }
    if let bInt = Int(b) {
        return "\(bInt)"
    }
    return "0"
}
